src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Properties/InterceptedProjectProperties/PackagePropertyPage/PackageIconValueProvider.cs

```diff
@@ -45,29 +45,28 @@ internal sealed class PackageIconValueProvider : InterceptingPropertyValueProvid
         }
 
         // https://docs.microsoft.com/en-us/nuget/reference/msbuild-targets#packageicon
-        private static string CreatePackageIcon(string filePath, string? packagePath)
+        private static string CreatePackageIcon(string filePath, string packagePath)
         {
             string filename = Path.GetFileName(filePath);
             // Make a slash-only value into empty string, so it won't get prepended onto the path.
-            // Also, make sure that the packagePath is non-null, so Path.Combine doesn't throw an ArgumentNullException.
-            if (packagePath is null || "\\".Equals(packagePath, StringComparisons.Paths) || "/".Equals(packagePath, StringComparisons.Paths))
+            if ("\\".Equals(packagePath, StringComparisons.Paths) || "/".Equals(packagePath, StringComparisons.Paths))
```

Comment:
Null check here was not necessary. Every call to this method will provide either a value or `string.Empty` for `packagePath`.